image_publish_topic: 'image_raw'

balls_topic: 'balls_in_image'
obstacles_topic: 'obstacles_in_image'
lines_topic: 'line_in_image'
goalposts_topic: 'goal_parts_in_image'
field_boundary_topic: 'field_boundary_in_image'

line_thickness: 4
